Update debugging.rst (#6326)
cakephp/docs: en/development/debugging.rst

.. php:class:: Debugger

> To use the debugger, in your app/Config.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.

Reviewer: There is no such file. I am against making this too concrete. Other files like bootstrap could also set this. No need to overspecify.

Author: adjusted. I changed the names. Thank you

> To use the debugger, in your config/app.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.

Reviewer 2: even in the `bootstrap.php` file you can set `Configure::read('debug')` to `true`
and even you can dont have a `config/app.php` 
so , above context 
```
To use the debugger, first ensure that ``Configure::read('debug')`` is
set to ``true``.
```
is more useful , you can keep it as it was ,and add your context after it , 
```
......also you can set `debug` in config/app.php file .......
```

>     Production Mode: set false. No error messages, errors, or warnings shown. Flash messages redirect.

Reviewer: Flash message part is incorrect. It can be side effect but in most cases this is not the case.

Author: > Especially the debug mode part. So best never to mention this.

Why @dereuromark ?

Reviewer: Because it is both incorrect and not needed info.

.. php:class:: Debugger

To use the debugger, first ensure that ``Configure::read('debug')`` is
set to ``true``.

> You can use ``filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` in **config/app.php** file to ensure that ``debug`` is a boolean.

Reviewer: @markstory I dont think the sentence was meant like this, or should be merged as such. In the scope of this topic it doesn't bear any useful info.

Reviewer: Refs my clear objection @ https://github.com/cakephp/docs/pull/6326#pullrequestreview-332268676

> Also you can set `debug` in config/app.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.

Reviewer 3: ```suggestion
You can use ``filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` in **config/app.php** file to ensure that ``debug`` is a boolean.
```

Author: thanks you
